### DISCUSSION TITLE
Networking error with ssh, ping, etc.

### DISCUSSION THREAD

**Discussion by florianwalter** [ghostty-org/ghostty#3819] [Feature Requests, Ideas] [answered]
I can't use ssh, ping etc. in Ghostty with my local machines.

E.g. when i ping a machine, i get a _"ping: sendto: No route to host"_. Same for ssh.
But traceroute seems to work.

<img width="797" alt="Ghostty-networking" src="https://github.com/user-attachments/assets/aa3672a1-d175-49e3-936b-59401c8f26ba" />

When i do the same in iTerm2, it works.

I am on macOS 15.2 and my shell is zsh.

**Comment by jmckillican** [comment, marked as answer]
Check if Ghostty has network access, you might have accidentally blocked or denied its access?   System Settings -> Privacy & Security -> Local Network

Also check if the Apple Firewall is on, possibly try turning it off and see if that helps.  System Settings -> Network -> Firewall

**Reply by florianwalter** [reply]
Thanks! 
That's embarrassing... local network access was disabled.
